packages/devtools_app/lib/src/flutter/scaffold.dart

```diff
@@ -45,6 +47,14 @@ class DevToolsScaffold extends StatefulWidget {
   /// The size that all actions on this widget are expected to have.
   static const double actionWidgetSize = 48.0;
 
+  /// The border around the content in the DevTools UI.
+  static const EdgeInsets borderInsets = EdgeInsets.only(
```

Comment:
also maybe rename to appPadding or similar? Border suggests that there is an actual border outline

Reply:
@pq this is a P3 lint to nudge users to use the more idiomatic constructor for EdgeInsets. This quick fix would be trivial.

Reply:
(renamed to `appPadding`)